src/components/video-container/subtitles.ts

```diff
@@ -35,8 +35,8 @@ const videoTextTtracksManager = (video: HTMLVideoElement, hls: Hls) => {
   };
 
   const tracksToStoreState = () => {
-    const pageLanguage = document.documentElement.lang;
-    const pageLanguageCode = pageLanguage.split('-')?.[0];
+    const userLanguage = navigator.language;
+    const userLanguageCode = userLanguage.split('-')[0];
```

Comment:
let's add here optional chaining operator jic `userLanguage?split`

Reply:
@timaramazanov awesome, I'll merge it then since I added two fallbacks:
```js
const userLanguage = navigator.language || document.documentElement.lang || 'en';
```